src/functions/createStoryMetaSettings.js

```diff
@@ -109,13 +109,24 @@ export function createStoryMetaSettings({
 
   actionPropsArray?.forEach(actionProp => {
     if (typeof actionProp === 'string') {
-      argTypes[actionProp] = { action: actionProp };
+      argTypes[actionProp] = { action: actionProp, control: false };
     } else if (actionProp?.name && actionProp.linkedToPropValue) {
       // we assume that actionPropsArray is static. If it changes, things may break, since internally we call React.useState for the story decorator.
       decorators.push(createMappedActionToInputPropDecorator(actionProp.name, actionProp.linkedToPropValue));
     }
   });
 
+  // If prop type is ElementContent, set text control, otherwise it's displaying as object
+  const componentProps = component?.__docgenInfo?.props;
```

Comment:
I think there's a nicer way of getting the props from SB, maybe something like `useDocsContext`, can't remember exactly

Reply:
Haven't found smth like this :(
Anyway it's a temp solution for storybook 6, so I believe it's fine